### DISCUSSION TITLE
Having some issues

### DISCUSSION THREAD

**Discussion by mphemsing** [LeDragoX/Win-Debloat-Tools#181] [Q&A]
I am using a Surface Pro 8 with Windows 11 Pro Version 23H2 Installed on ‎9/‎5/‎2024 OS build 22631.4112 Windows Feature Experience Pack 1000.22700.1034.0

After applying tweaks I am now unable to right click program icons on the taskbar regardless of if they are opened or pinned. I am also unable to type in the search bar of the start menu.

Any ideas or known fixes?

**Comment by mphemsing** [comment]
got the search issue fixed by enabling search

still unable to right click on taskbar programs

**Reply by LeDragoX** [reply]
Can you click this option twice? The first click will enable, but the second will disable the Legacy Context Menu.

![image](https://github.com/user-attachments/assets/db766ca6-b791-4020-b367-f1f4ae524ea0)

People have been trying to run the script on Tablets and unexpected bugs are happening.
